airflow/providers/google/cloud/operators/datafusion.py

```diff
@@ -674,14 +676,23 @@ def execute(self, context: Dict):
             project_id=self.project_id,
         )
         api_url = instance["apiEndpoint"]
-        hook.start_pipeline(
+        faux_pipeline_id = hook.start_pipeline(
             pipeline_name=self.pipeline_name,
             instance_url=api_url,
             namespace=self.namespace,
             runtime_args=self.runtime_args
 
         )
         self.log.info("Pipeline started")
+        hook.wait_for_pipeline_state(
+            success_states=[PipelineStates.COMPLETED],
```

Comment:
This operator is name `CloudDataFusionStartPipelineOperator` (key word start).
IMO that means `PipelineStates.RUNNING` should be a success state by default.
However, it might be worth allowing the user to optionally control these success states to span more use cases.

Reply:
I've added `success_states` as an optional argument. If not provided the operator will wait for operation to be `RUNNING`